client/app/cdn/dedicated/backend/order/cdn-dedicated-backend-order.html

```diff
@@ -1,123 +1,117 @@
+                                </span>
+                            </oui-field>
+                        </oui-step-form> <!-- End of Step 1: Choose quantity -->
+
+                        <!-- Step 2: Choose duration -->
+                        <oui-step-form data-header="{{ :: 'cdn_dedicated_backend_order_step_2_title' | translate }}"
+                                       data-description="{{ $ctrl.model.backendQuantity > 1 ? 'cdn_dedicated_backend_order_step_2_description_plural' : 'cdn_dedicated_backend_order_step_2_description' | translate:{ quantity: $ctrl.model.backendQuantity } }}"
+                                       data-loading="$ctrl.loading.durations"
+                                       data-loading-text="{{ :: 'cdn_dedicated_backend_order_step_2_loading' | translate }}"
+                                       data-valid="$ctrl.model.duration">
+
+                                <div class="row mb-4">
+                                    <div class="col-xs-3"
+                                         data-ng-repeat="price in $ctrl.prices track by $index">
+                                        <oui-radio data-text="{{:: price.duration.text }}"
+                                                   data-id="duration-{{ price.duration.value }}"
+                                                   data-value="price"
+                                                   data-thumbnail
+                                                   data-name="duration"
+                                                   data-model="$ctrl.model.duration">
+                                            <span class="oui-radio__section"
+                                                  data-translate="price_ht_label"
+                                                  data-translate-values="{ price: price.prices.withoutTax.text }">
+                                            </span>
+                                        </oui-radio>
+                                    </div>
+                                </div>
+
+                        </oui-step-form> <!-- End of Step 2: Choose duration -->
+
+                        <!-- Step 3: Contracts accept -->
+                        <oui-step-form data-header="{{ :: 'cdn_dedicated_backend_order_step_3_title' | translate }}"
+                                       data-valid="$ctrl.model.contracts">
+                            <div data-ng-repeat="contract in $ctrl.model.duration.contracts track by $index">
+                                <a target="_blank"
+                                   rel="noopener"
+                                   class="oui-link oui-link_icon"
+                                   data-ng-href="{{ contract.url }}">
+                                    <span data-ng-bind="contract.name"></span>
+                                    <span class="oui-icon oui-icon-external_link"
```

Comment:
There should be a text wi `class="sr-only"` to tell the users the link will open in a new tab